src/main/kotlin/eu/kevin/api/v03/payments/InitiatePaymentRequest.kt

```diff
@@ -0,0 +1,61 @@
+package eu.kevin.api.v03.payments
+
+import java.math.BigDecimal
+import java.time.LocalDate
+
+data class InitiatePaymentRequest @JvmOverloads constructor(
```

Comment:
I think we should discuss which is the better way for you guys for placing classes. The way usually I do, I would create a separate package in the payments called initiate and then place all classes there in the separate files.